### ISSUE TITLE
"make install" missing

### ISSUE THREAD

**Issue by bnewbold** [jvasile/Plinth#4]
There is only a "fabric" mechanism for system-wide installation.

Also, unless i'm just not seeing that which is right in front of me, there is no mechanism for building a debian package.

**Comment by jvasile** [comment]
Yeah, I think I forgot to make a deb package for that.  You're right.

bnewbold
reply@reply.github.com
writes:

**Comment by petterreinholdtsen** [comment]
http://git.tzafrir.org.il/?p=plinth/plinth.git and http://tzafrir.org.il/~tzafrir/fbox/ contain example deb packaging for plinth.
